packages/patternfly-4/react-core/src/components/Page/Page.tsx

```diff
@@ -1,56 +1,52 @@
-import React, { cloneElement } from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Page/page';
 import { css } from '@patternfly/react-styles';
 import { global_breakpoint_md as globalBreakpointMd } from '@patternfly/react-tokens';
 import { debounce } from '../../helpers/util';
 
-export const PageLayouts = {
-  vertical: 'vertical',
-  horizontal: 'horizontal'
+export enum PageLayouts {
+  vertical = 'vertical',
+  horizontal = 'horizontal'
 };
 
-const propTypes = {
-  /** Content rendered inside the main section of the page layout (e.g. <PageSection />) */
-  children: PropTypes.node,
-  /** Additional classes added to the page layout */
-  className: PropTypes.string,
-  /** Header component (e.g. <PageHeader />) */
-  header: PropTypes.node,
-  /** Sidebar component for a side nav (e.g. <PageSidebar />) */
-  sidebar: PropTypes.node,
-  /** Skip to content component for the page */
-  skipToContent: PropTypes.node,
-  /** Breadcrumb component for the page */
-  breadcrumb: PropTypes.node,
-  /** If true, manages the sidebar open/close state and there is no need to pass the isNavOpen boolean into
-   * the sidebar component or add a callback onNavToggle function into the PageHeader component */
-  isManagedSidebar: PropTypes.bool,
-  /** Can add callback to be notified when resize occurs, for example to set the sidebar isNav prop to false for a width < 768px
-   * Returns object { mobileView: boolean, windowSize: number } */
-  onPageResize: PropTypes.func,
-  /** Additional props are spread to the container <div> */
-  '': PropTypes.any // eslint-disable-line react/require-default-props
-};
+export interface PageProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
+  className?: string;
+  header?: React.ReactNode;
+  sidebar?: React.ReactNode;
+  skipToContent?: React.ReactElement;
+  isManagedSidebar?: boolean;
+  onPageResize?: (object: object) => void;
```

Comment:
We use `any` instead of `object`.

Reply:
fixed

Reply:
I don't see `(object: any) => void` in your latest commit f8de73a1567767e10222ce799998236af3ea410a .